metapool/sample_sheet.py

```diff
@@ -803,6 +816,119 @@ def __init__(self, path=None):
         }
 
 
+class MetagenomicSampleSheetv101(KLSampleSheet):
+    # Adds support for optional KATHAROSEQ columns in [Data] section.
+
+    _HEADER = {
+        'IEMFileVersion': '4',
+        'SheetType': _STANDARD_METAG_SHEET_TYPE,
+        'SheetVersion': '101',
+        'Investigator Name': 'Knight',
+        'Experiment Name': 'RKL_experiment',
+        'Date': None,
+        'Workflow': 'GenerateFASTQ',
+        'Application': 'FASTQ Only',
+        'Assay': _METAGENOMIC,
+        'Description': '',
+        'Chemistry': 'Default',
+    }
+
+    data_columns = ['Sample_ID', 'Sample_Name', 'Sample_Plate', 'well_id_384',
+                    'I7_Index_ID', 'index', 'I5_Index_ID', 'index2',
+                    'Sample_Project', 'Well_description']
+
+    # Names of columns needed in the sample-information-file output that is
+    # later passed into Qiita:
+    # 'katharoseq_aliquot_volume_ul',
+    # 'katharoseq_batch_information',
+    # 'katharoseq_cell_count',
+    # 'katharoseq_plate_number',
+    # 'katharoseq_strain'
```

Comment:
If no need, can you delete these comments? For example: katharoseq_strain is not in any other part of this PR.